geth/jail/internal/timers/timers_test.go

```diff
@@ -91,6 +91,29 @@ func (s *TimersSuite) TestClearIntervalImmediately() {
 	<-time.After(100 * time.Millisecond)
 }
 
+func (s *TimersSuite) TestImmediateTimer() {
```

Comment:
I wonder when `setImmediate` can actually return an error... I guess it's not really possible to write a test case? Btw. what's is `v` value when printed? `[Object object]`?

Reply:
If I remember well it was possible make `call.Otto.ToValue` to fire an error by passing a nil value. However, I considered implementing the logic to fire it need it heavy some changes on the code, and considering testing the panic is only a trivial line (only to increment coverage) I didn't went for it. Additionally I didn't saw a clear way to mock it, without many changes.

You're right `v` was printing `[Object object]`. I was trying to dump object properties/methods, but all of them were empty. I'm not really familiar with `otto.Value` but I'm open to suggestions if you want to check some specific property :)

Reply:
Nah, I think it's good. I was just curious if you found an easy way to do it.